README.md

```diff
@@ -350,6 +349,20 @@ for details and use-cases.
     Alice     = "Bob"
   }
   ```
+#### Cognito User Pool Domain
+
+- **`domain_prefix`**: *(Optional `string`)*
+
+  Type a domain prefix to use for the sign-up and sign-in pages that are hosted by Amazon Cognito,
+  e.g. `https://{YOUR_PREFIX}.auth.eu-west-1.amazoncognito.com`. The prefix must
+  be unique across the selected AWS Region. Domain names can only contain
+  lower-case letters, numbers, and hyphens. Default is `""`.
```

Comment:
this is only one use case... if no certificate arn is givin or this is not a fqdn ... also this argument should be domain

Reply:
resolved in 3b4e65b